### ISSUE TITLE
corrupt energy consumption reading

### ISSUE THREAD

**Issue by herm42** [domoticz/domoticz#6190]
zwave-mqtt/116/50/0/value/65537

{"time":1729865646395,"value":4514.869,"nodeName":"Powermeter","nodeLocation":"Zählerkasten"}

![image](https://github.com/user-attachments/assets/20eb0dc3-b04d-47bb-83c2-250f25db3bef)

![image](https://github.com/user-attachments/assets/bfd9f37f-dc92-469f-86e5-9e2b800374dc)

the reading is correct every new day but turn completely wrong around mid day...

this behaviour started on beta versions beginning sept 19th

![image](https://github.com/user-attachments/assets/a0552635-c0d8-4af0-9bea-5e7a5620adb5)

according to the mqtt message the actual value should be 4514.869 ... 

... before the reading where correct

{
  "value_template": "{{ value_json.value }}",
  "unit_of_measurement": "kWh",
  "state_class": "total_increasing",
  "device_class": "energy",
  "state_topic": "zwave-mqtt/116/50/0/value/65537",
  "availability": [
    {
      "payload_available": "true",
      "payload_not_available": "false",
      "topic": "zwave-mqtt/116/status",
      "value_template": "{{'true' if value_json.value else 'false'}}"
    },
    {
      "topic": "zwave-mqtt/_CLIENTS/ZWAVE_GATEWAY-zwave-js-ui/status",
      "value_template": "{{'online' if value_json.value else 'offline'}}"
    },
    {
      "payload_available": "true",
      "payload_not_available": "false",
      "topic": "zwave-mqtt/driver/status"
    }
  ],
  "availability_mode": "all",
  "json_attributes_topic": "zwave-mqtt/116/50/0/value/65537",
  "device": {
    "identifiers": [
      "zwavejs2mqtt_0xe9a1d9f4_node116"
    ],
    "manufacturer": "AEON Labs",
    "model": "Home Energy Meter - Gen5 (ZW095)",
    "name": "Powermeter",
    "sw_version": "1.6"
  },
  "name": "Powermeter_electric_kwh_value",
  "unique_id": "zwavejs2mqtt_0xe9a1d9f4_116-50-0-value-65537"
}

zwave/sensor/Powermeter/electric_kwh_value/config

**Comment by gizmocuz** [comment]
@dwmw2 , I am pretty sure this is related to your patch on 19 September
I have created a PR to revert this and merged it

https://github.com/domoticz/domoticz/pull/6191

I am going to add support for the "kw" unit and fix the typos

**Comment by gizmocuz** [comment]
@herm42 , could you try beta 16296?

**Comment by herm42** [comment]
deployed beta 16296 ... but it terminates after a few seconds

2024-10-26T09:15:44.249138+02:00 ha domoticz: mosquitto: Discovered: CO Küche (CO Küche_temperature_air)/CO Küche (CO Küche_temperature_air) (unique_id: zwavejs2mqtt_0xe9a1d9f4_14-49-0-Air_temperature)
2024-10-26T09:15:44.349941+02:00 ha domoticz: mosquitto: Discovered: CO Küche (CO Küche_gas_carbon_monoxide)/CO Küche (CO Küche_gas_carbon_monoxide) (unique_id: zwavejs2mqtt_0xe9a1d9f4_14-49-0-Carbon_monoxide_CO_level)
2024-10-26T09:15:44.354557+02:00 ha systemd[1]: domoticz.service: Main process exited, code=killed, status=11/SEGV
2024-10-26T09:15:44.354595+02:00 ha systemd[1]: domoticz.service: Failed with result 'signal'.

**Comment by herm42** [comment]
always at exact the same point ...
2024-10-26T09:16:50.227631+02:00 ha domoticz: mosquitto: Discovered: CO Küche (CO Küche_gas_carbon_monoxide)/CO Küche (CO Küche_gas_carbon_monoxide) (unique_id: zwavejs2mqtt_0xe9a1d9f4_14-49-0-Carbon_monoxide_CO_level)
2024-10-26T09:16:50.231357+02:00 ha systemd[1]: domoticz.service: Main process exited, code=killed, status=11/SEGV

**Comment by gizmocuz** [comment]
Whoops, sorry about that. Corrected the issue, please try beta 16297

**Comment by herm42** [comment]
better now :-) ... 
again: thx for effort!

**Comment by gizmocuz** [comment]
Great, hope it keeps working, please let us know

**Comment by dwmw2** [comment]
> @dwmw2 , I am pretty sure this is related to your patch on 19 September I have created a PR to revert this and merged it
> 
> #6191
> 
> I am going to add support for the "kw" unit and fix the typos

Makes sense. Clearly HA isn't following its own documentation for `total_increasimg`. Or it works around certain sensors which say `total_increasing` when they aren't really, with some heuristics. We need to work out what HA is doing before we turn the behaviour back on.

**Comment by dwmw2** [comment]
Perhaps we could leave the code in place, just disabled until we learn the precise conditions to enable it and be compatible with HA? Better still if there's a way for users who have a genuinely `total_increasing` device to enable it.

**Comment by waltervl** [comment]
> better now :-) ... again: thx for effort!

Closing this
